velox/common/encode/Base64.h

```diff
@@ -139,7 +143,8 @@ class Base64 {
   // character.
   static uint8_t base64ReverseLookup(
       char encodedChar,
-      const ReverseIndex& reverseIndex);
+      const ReverseIndex& reverseIndex,
+      Status& status);
```

Comment:
Status should always be the return type. The return value is not valid if Status is not OK. You can combine Status with a return value using folly::Expected.
See https://github.com/facebookincubator/velox/pull/7589/
Here I think you can return `uint8_t` as a return argument.

Reply:
Updated the code in a similar way.